climada/engine/test/test_impact.py

```diff
@@ -467,6 +467,48 @@ def test_local_exceedance_imp_pass(self):
         self.assertAlmostEqual(np.max(impact_rp), 2916964966.388219, places=5)
         self.assertAlmostEqual(np.min(impact_rp), 444457580.131494, places=5)
 
+class TestImpactReg(unittest.TestCase):
+    """Test impact aggregation per exposure region id or admin0"""
+    def test_impact_at_reg(self):
+        """Test calc local impacts per region"""
+
+        # Read default hazard file
+        hazard = Hazard.from_hdf5(HAZ_TEST_TC)
+
+        # Read an exposure
+        ent = Entity.from_excel(ENT_DEMO_TODAY)
+        ent.check()
+
+        # Calculate impact
+        impact = ImpactCalc(ent.exposures, ent.impact_funcs, hazard).impact(save_mat=True)
```

Comment:
I would much prefer if you could create a impact object from scratch here. All you need for this test is a very simple impact matrix (e.g., 2 events x 4 coords) and corresponding exposure coordinates (in this case, 4). Then you can check for the exact values you put into the matrix further below, instead of these seemingly random numbers based on the example data. If you get stuck please notify me and I'll give it a try.

Reply:
Tests have been rewritten. I am now using the dummy impact object also used in other impact tests.